lib/model/list_model.dart

```diff
@@ -91,32 +85,8 @@ class ListModel {
   /// Gets the `Item` stored in `IsarLinks` with `item`'s id.
   Item lookup(Item item) => items.lookup(item)!;
 
-  void reorderItem({required int oldOrder, required int newOrder}) {
-    assert(oldOrder < items.length && oldOrder >= 0);
-    assert(newOrder < items.length && newOrder >= 0);
-
-    int lower = min(oldOrder, newOrder);
-    int upper = max(oldOrder, newOrder);
-
-    final itemsToReorder = items
-        .filter()
-        .orderBetween(lower, upper)
-        .findAllSync()
-        // lookup all the items so that the new orderings are instantly/synchronously reflected
-        // in the `IsarLinks` copy of the database items.
-        .map((item) => lookup(item));
-    // note: we can use `findFirst` since there should be exactly one
-    // element ordered with `oldOrder`.
-    final itemWithOldOrder =
-        lookup(items.filter().orderEqualTo(oldOrder).findFirstSync()!);
-    final reorderingOffset = (oldOrder - newOrder).sign;
-
-    for (final item in itemsToReorder) {
-      item.order += reorderingOffset;
-    }
-
-    itemWithOldOrder.order = newOrder;
-    DatabaseManager.putItems([...itemsToReorder, itemWithOldOrder]);
+  reorderItems(List<Item> items) async {
+    await DatabaseManager.putItems(items);
```

Comment:
Two small styling questions:
- Do you think we should add a return type to `reorderItems` (otherwise it appears as `dynamic` in the IDE)?
- Would there be any disadvantage to making this an "arrow" function (one line)?

Reply:
Great points!